build.gradle

```diff
@@ -16,12 +16,7 @@ contacts {
 dependencies {
     // TODO, make these optional
 
-    // Specifically not using com.perforce:p4java since it has a META-INF/INDEX.LIST file that lists
-    // com.jcraft. In additional to wrongness of bundle thirdparty libraries, it doesn't even have 
-    // those class files. The classloader will consider the INDEX.LIST file depending on ordering
-    // of the classpath, so it's very likely that just havine p4java in your path will prevent the
-    // JVM from even running.
-    compile 'com.perforce:p4java-jfrog:2011.1.297684'
+    compile 'com.perforce:p4java:2014.1.965322'
```

Comment:
Seems like there was a reason for not using `p4java` and instead using `p4java-jfrog`. @quidryan Can you comment on this pull request?